Dockerfile.assisted-service-build

```diff
@@ -34,10 +34,10 @@ COPY --from=quay.io/openshift/origin-cli:latest /usr/bin/oc /usr/bin
 COPY --from=quay.io/operator-framework/upstream-opm-builder:v1.16.1 /bin/opm /bin
 COPY --from=k8s.gcr.io/kustomize/kustomize:v4.3.0 /app/kustomize /usr/bin/
 
-RUN dnf install -y 'dnf-command(config-manager)' && \
+RUN dnf install -y 'dnf-command(config-manager)' epel-release && \
```

Comment:
be aware we previously had some availability issues with epel
although I don't see other way to install ShellCheck

Reply:
Hmmm looks like `quay.io/coreos/shellcheck-alpine:v0.5.0` which I originally avoided because I thought it would expect musl is actually statically linked, so we can grab it from that quay image, and even get version pinning for free

Reply:
Sadly v0.5.0 is pretty old, you win some you lose some

Reply:
Consider that each `COPY --from` would download the full image to the disk. At the moment a full build for the build image is taking 15GB of space in our local env machines (maybe only about 1 or 2GB are the actual output).

Reply:
You win some you lose even more? 

But seriously the alpine image is only 25MB so I guess it's not too bad